scripts/network.js

```diff
@@ -182,27 +182,51 @@ export class ExplorerNetwork extends Network {
         return this.blocks;
     }
 
+    async safeFetchFromExplorer(strCommand) {
+        let trials = 0;
+        const maxTrials = 5;
+        while (trials < maxTrials) {
+            try {
+                trials += 1;
+                const res = await fetchBlockbook(strCommand);
+                if (!res.ok) throw res;
```

Comment:
Don't throw res, the catch can probably be removed altogether, since it's not the type of error you wanna catch (fetch doesn't throw if the server returns an error)
This also needs some comments